tools/prepublish.ts

```diff
@@ -4,8 +4,7 @@
  * Distributed under MIT license.
  * See file LICENSE for detail or copy at https://opensource.org/licenses/MIT
  */
-
-import * as async from 'async';
+import {promisify} from '@google-cloud/promisify';
```

Comment:
Just `import {promisify} from 'util'` will do just fine here.

Reply:
@JustinBeckwith  Done, but it's failing for node6

Reply:
🤦‍♂️  I'm sorry.  I didn't realize the unit tests would be covering this directory for some reason.  Feel free to use the previous code.  Again, apologies for making you change it back.